### PR TITLE
Bartender Drip DLC

### PR DESCRIPTION
**Pull request by Golinth**
## About the PR

Added some amazing bartender drip made by sunnysmiles11 (Lena Milize) on discord, added the new clothes to loadouts, and some new senior bartender drip as well (As explained by Lena, experienced bartenders IRL wear black clothes). The timegate can be removed if it is a point of contention.

## Why / Balance

Additional bartender drip. Shouldn't affect game balance, as it is drip. With the dark uniform, the crew can expect the bartender to be experienced at their job, and may feel comfortable ordering more advanced drinks.

## Technical details

Purely YML and sprite changes.

## Media

image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added new bartender drip!

### CODE REVIEW

**Comment by lzk228** [comment]
wouldnt be better to make bowtie a separate item?

**Comment by skypatrol21** [comment]
Much-needed diversity to an underappreciated role this addition would help add a little variety to one of the most RP heavy roles in the game. Up and tell this point, bartender drip has been pretty limited. This would be a refreshing update.

**Comment by UbaserB** [comment]
I LOVE TIME-GATES!
Honestly, I feel like we need more time-gates for all sorts of roles, even if it's not "senior". It helps with the immersion and gives you goals to work towards. Always appreciate that.

**Comment by Golinth** [comment]
> wouldnt be better to make bowtie a separate item?

Probably yes, but that would add a little bit more work than I wanted on this PR, and I was hoping to get this merged right before the content freeze takes place. After the content freeze I'm 100% open to adding some new bowtie options as an alternative to neckties.

**Comment by EmoGarbage404** [comment]
i don't really see a point in adding playtime gating to these

File: Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 1]
```diff
@@ -31,6 +31,28 @@
...
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitBowtieBartender
+  name: bartender's red bowtie uniform
+  description: A nice and tidy uniform, but this time with a red bowtie, nice and snazzy.
+  components:
+  - type: Sprite
+    sprite: Clothing/Uniforms/Jumpsuit/bartender_bowtie.rsi
+  - type: Clothing
+    sprite: Clothing/Uniforms/Jumpsuit/bartender_bowtie.rsi
+
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml:55]: Why can't we just use recolouring for this? Seems unnecessarily copy-pasted together.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

**Comment by Golinth** [comment]
> i don't really see a point in adding playtime gating to these

Only the dark color scheme gets the playtime gate, and the reasoning behind it was that IRL, experienced bartenders do wear darker colors.

**Comment by Golinth** [comment]
Mobile GitHub moment. I’ll start working on the review.

**Comment by EmoGarbage404** [comment]
Closed due to feature freeze May 10th-June 14th. Comment to have it reopen after this.

**Pull request closed without merging**